packages/snowpack/src/config.ts

```diff
@@ -230,24 +231,28 @@ function loadPlugins(
   function loadPluginFromConfig(name: string, options?: any): SnowpackPlugin {
     const pluginLoc = require.resolve(name, {paths: [process.cwd()]});
     const pluginRef = require(pluginLoc);
-    const plugin = pluginRef.default
-      ? pluginRef.default(config, options)
-      : pluginRef(config, options);
+    let plugin: SnowpackPlugin & LegacySnowpackPlugin;
```

Comment:
Added this legacy plugin type. There was too much going on here where a simple mistake seemed inevitable.